packages/dds/merge-tree/src/client.ts

```diff
@@ -292,6 +292,20 @@ export class Client {
             accum, start, end, splitRange);
     }
 
+    public getGCData(handle: IFluidHandle, handleCollectingSerializer: IFluidSerializer): void {
```

Comment:
One thing that we can do to optimize it a bit is to leverage previously collected (during snapshot generation) GC data IF msn === current sequence number.
I'd expect that to be the case (all the time) if there is only one user working on the doc.
That said, based on what I heard from test run, it's almost always not the case (otherwise we would not see these problems in single-session test pass), and I wonder - why?

Reply:
Hm, do we run summary twice anyway (always)? If so, than my comment is about nothing

Reply:
yeah, we do run it twice today. And getGCData runs before summarize does.